x/gov/keeper/proposal_test.go

```diff
@@ -244,6 +288,13 @@ func (suite *KeeperTestSuite) TestCancelProposal() {
 			}
 		})
 	}
+	_, err = suite.govKeeper.Votes.Get(suite.ctx, collections.Join(proposalID, suite.addrs[0]))
+	suite.Require().ErrorContains(err, collections.ErrNotFound.Error())
```

Comment:
here we check the votes are not found for proposal 1 as it executes CancelProposal